### PR TITLE
Eslint changes

### PR DESCRIPTION
**Pull request by glidej**
- import React explicitly in files that use JSX - [react-in-jsx-scope](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/react-in-jsx-scope.md)
- validate props via propTypes (needed to install `prop-types` package to support without deprecation warning) - [prop-types](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prop-types.md)
- set display name - [display-name](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/display-name.md)
- various spacing issus

### CODE REVIEW

File: components/footer.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
+import React from 'react'
 import Link from 'next/link'
 
-export default () => (
+const Footer = () => (
```
Comment (glidej) [inline components/footer.js:4]: Moving to the `const Component` style was a bit of a refactor to support displayName and propTypes. It seemed less destructive then converting everything to React.Component and jamming those things in the constructor.

File: package.json [hunk 2]
```diff
@@ -15,6 +15,7 @@
     "axios": "^0.16.0",
     "moment": "^2.18.1",
     "next": "^2.0.1",
+    "prop-types": "^15.5.7",
```
Comment (glidej) [inline package.json:18]: react 15.5 deprecates using `React.PropTypes`, they recommend using the `prop-types` package.

**Comment by sixlive** [comment]
Resolves #11

**Comment by sixlive** [comment]
@glidej Awesome work!! Thank you for all the contributions!

**Pull request merged**